apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/GetObjectOptions.cs

```diff
@@ -30,6 +30,11 @@ public sealed class GetObjectOptions
         /// </summary>
         public long? Generation { get; set; }
 
+        /// <summary>
+        /// If true, only soft-deleted object versions will be listed. The default is false.
```

Comment:
We're not listing, just getting, so this should probably be "only soft-deleted object versions will be retrieved" or similar.